core/src/main/scala/kafka/utils/Time.scala

```diff
@@ -44,6 +46,8 @@ trait Time {
 
   def nanoseconds: Long
 
+  def hiResClockMs: Long = TimeUnit.NANOSECONDS.toMillis(nanoseconds)
```

Comment:
Why have we implemented this here instead of SystemTime with a different implementation for MockTime?